package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A service responsible for handling upload requests",
   "main": "lib/index.js",
   "scripts": {
-    "build": "babel --extensions '.ts,.tsx' src -d lib",
+    "build": "node ./node_modules/@babel/cli/bin/babel.js --extensions '.ts,.tsx' src -d lib",
```

Comment:
Windows won't allow single quotes in  ` '.ts, .tsx'`.. So change it

Reply:
Oh right!  Good call -- fixing it up now.